book/src/super-sql/operators/aggregate.md

```diff
@@ -47,7 +47,7 @@ Each aggregate function `<agg-func>` may be optionally followed by a `where` cla
 which applies a Boolean expression `<expr>` that indicates, for each input value,
 whether to include it in the values operated upon by the aggregate function.
 `where` clauses are analogous
-to the [`where`](where.md) operator but apply their filter to the input
+to the [`where` operator](where.md) but apply their filter to the input
```

Comment:
Edits like this one are to maintain a prior convention that when we found ourselves hyperlinking to specific operators/functions, etc., we'd try to include the word "operator"/"function"/etc. and also put it inside the hyperlinked text. The thinking has been that this would help disambiguate in the small number of cases where we've got the same keyword for two different things, e.g., "`fuse` operator" and "`fuse` aggregate function".